tests/regress/sql/test_activetable_limit.sql

```diff
@@ -19,29 +20,34 @@ INSERT INTO a03 values(generate_series(0, 500));
 
 \c test_tablenum_limit_02
 CREATE EXTENSION diskquota;
+
+CREATE EXTERNAL WEB TABLE segment_logs(line text)
+    EXECUTE 'cat $GP_SEG_DATADIR/pg_log/$(ls -Art $GP_SEG_DATADIR/pg_log | tail -n 1)'
+    ON ALL FORMAT 'TEXT' (DELIMITER 'OFF');
+
 CREATE SCHEMA s;
 SELECT diskquota.set_schema_quota('s', '1 MB');
 
 SELECT diskquota.wait_for_worker_new_epoch();
 
-CREATE TABLE s.t1(i int) DISTRIBUTED BY (i); -- activetable = 1
-INSERT INTO s.t1 SELECT generate_series(1, 100000); -- ok. diskquota soft limit does not check when first write
+-- We create twice as many tables as the limit to ensure that the active_tables table is overflow.
+CREATE TABLE s.t1 (a int, b int) DISTRIBUTED BY (a)
+    PARTITION BY RANGE (b) ( START (0) END (10) EVERY (1) );
```

Comment:
May we suppress notices?

Reply:
these notices should already be suppressed.
https://github.com/arenadata/diskquota/blob/5e402a7475cc6bfaee295ea0f49a0c3e753d2d06/tests/init_file#L10

Reply:
see tests/regress/expected/test_activetable_limit.out

Reply:
The test_many_active_table and test_partion tests also have these notices.

Reply:
you may increase log level to warning before create table with partitions and reset after

Reply:
I don't see the point in doing this since all the testers leave these notices as is.